client/components/theme/index.jsx

```diff
@@ -99,7 +99,10 @@ export class Theme extends Component {
 	}
 
 	onScreenshotClick = () => {
-		this.props.onScreenshotClick( this.props.theme.id, this.props.index );
+		const { onScreenshotClick } = this.props;
+		if ( typeof onScreenshotClick === 'function' ) {
+			this.props.onScreenshotClick( this.props.theme.id, this.props.index );
```

Comment:
nit: can you use the destructured `onScreenshotClick` variable when doing the actual call.

Also, I'd personally just check for `if ( onScreenshotClick ) { /* call it */ }`, but that's a matter of taste 🙂

Reply:
> nit: use the destructured `onScreenshotClick` variable when doing the actual call.

uh right, the fact that i need it twice was the sole reason i destructured it in the first place :laughing: 

> Also, I'd personally just check for `if ( onScreenshotClick ) { /* call it */ }`, but that's a matter of taste 🙂 

Yeah, I was considering that, since we have the `propType` specified. But like the console error this was causing before, I thought it'd might be easy to miss. Inclined to stick with the `typeof` check.